packages/blockchain-wallet-v4-frontend/src/modals/SendCrypto/TransactionOverview/index.tsx

```diff
@@ -0,0 +1,191 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import { connect, ConnectedProps } from 'react-redux'
+import BigNumber from 'bignumber.js'
+import styled from 'styled-components'
+
+import { Button, SpinningLoader, Text } from 'blockchain-info-components'
+import CollapseText from 'components/CollapseText'
+import DataError from 'components/DataError'
+import CoinDisplay from 'components/Display/CoinDisplay'
+import FiatDisplay from 'components/Display/FiatDisplay'
+import { FlyoutWrapper, Row } from 'components/Flyout'
+import { AmountWrapper, StepHeader } from 'components/Flyout/SendRequestCrypto'
+import { selectors } from 'data'
+import { RootState } from 'data/rootReducer'
+
+import { Props as OwnProps } from '..'
+
+const Wrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  min-height: 100%;
+`
+const CustomRow = styled(Row)`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  > div:last-child * {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+  }
+`
+
+const TransactionOverview = (props: Props) => {
+  const { formValues, showNewSendFlow } = props
+  const { selectedAccount, to } = formValues
+  const { coin } = selectedAccount
+
+  return (
+    <Wrapper>
+      <FlyoutWrapper>
+        <StepHeader>
+          <Text size='24px' color='grey800' weight={600}>
+            <FormattedMessage id='modals.sendcrypto.enteramount.title' defaultMessage='Send' />
+          </Text>
+        </StepHeader>
+      </FlyoutWrapper>
+      {props.prebuildTxR.cata({
+        Failure: (e) => <DataError message={{ message: e }} />,
+        Loading: () => (
+          <div style={{ display: 'flex', justifyContent: 'center' }}>
+            <SpinningLoader width='14px' height='14px' borderWidth='3px' />
+          </div>
+        ),
+        NotAsked: () => (
+          <div style={{ display: 'flex', justifyContent: 'center' }}>
+            <SpinningLoader width='14px' height='14px' borderWidth='3px' />
+          </div>
+        ),
+        Success: (tx) => (
```

Comment:
what is the difference between transactionOverview and confirm?

Reply:
this is a new page last step of process, user do confirm and on this page we show again transaction detail and progress like number of confirmations + link to tx explorer